### ISSUE TITLE
[Bug]: JASP quits after opening file (corrupted file?) 

### ISSUE THREAD

**Issue by tvantimmeren** [jasp-stats/jasp-issues#1571]
### JASP Version

0.16

### What OS are you seeing the problem on?

macOS Mojave 10.14.6

### Bug Description

I have been working on some analyses yesterday, but now the file won't open again. What happens is that when I try to open the file, JASP starts loading the file but then quits without any further notice. 
My computer crashed (independent from JASP) before saving the JASP file, so this may have caused the issue. 

[CR_ROI_AUD_HC.jasp.zip](https://github.com/jasp-stats/jasp-issues/files/7853467/CR_ROI_AUD_HC.jasp.zip)
 

### Expected Behaviour

The file should just open :) 

### Steps to Reproduce

1. Open the file

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if any.
- [X] I have included a JASP file that causes the crash/bug, if any.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by boutinb** [comment]
[CR_ROI_AUD_HC2.jasp.zip](https://github.com/jasp-stats/jasp-issues/files/7853692/CR_ROI_AUD_HC2.jasp.zip)
Strange, I can open this file without problem. I have re-saved it and post it here. Can you try it?

**Comment by tvantimmeren** [comment]
No, still closes after it finishes loading the file. Very strange! 
I am able to open other files (some, not all), so it's not a generic JASP issue.

**Comment by boutinb** [comment]
After it crashes, can you generate the report from MacOS and send it to me? I see also that you use MacOS Mojave, but this version is not supported anymore by Apple, and so by JASP also. It's highly recommended to upgrade because you don't get any security patch anymore.

**Comment by tvantimmeren** [comment]
Hi Bruno
you're right, updating to the new macOS has been on my list for a while..
This was the final push I needed. It indeed solved the problem! Thanks for
your help.
Best,
Tim
